force-app/main/default/objects/Account_Soft_Credit__c/Account_Soft_Credit__c.object-meta.xml

```diff
@@ -60,6 +60,5 @@
     <pluralLabel>Account Soft Credits</pluralLabel>
     <searchLayouts/>
     <sharingModel>ControlledByParent</sharingModel>
-    <startsWith>Vowel</startsWith>
```

Comment:
These are set in the individual objectTranslation files, removing them from here allows deployment directly to non-english orgs.